src/main/java/com/ormi/mogakcote/user/presentation/UserController.java

```diff
@@ -0,0 +1,47 @@
+package com.ormi.mogakcote.user.presentation;
+
+import com.ormi.mogakcote.common.model.ResponseDto;
+import com.ormi.mogakcote.user.application.UserService;
+import com.ormi.mogakcote.user.dto.request.PasswordRequest;
+import com.ormi.mogakcote.user.dto.request.RegisterRequest;
+
+import com.ormi.mogakcote.user.dto.response.ValidatePasswordResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api")
+@RequiredArgsConstructor
+public class UserController {
+    private final UserService userService;
+
+
+    @GetMapping("/users/check-nickname")
+    public ResponseEntity<?> checkNickname(@RequestParam String username) {
+        var response = userService.checkNickname(username);
+        return ResponseDto.ok(response);
+    }
+
+    @GetMapping("/users/check-email")
+    public ResponseEntity<?> checkEmail(@RequestParam String email) {
+        var response = userService.checkEmail(email);
+        return ResponseDto.ok(response);
+
+    }
+
+    @PostMapping("/signup/validate-password")
```

Comment:
- 서비스 반환을 적용했고, 예외처리를 컨트롤러가 아닌 서비스에서 하도록 했습니다!
- 다른 코드에서도 마찬가지로 예외처리가 필요하다면 되도록 서비스에서 우선 작업해주시면 좋을 것 같아요.

```suggestion
       @PostMapping("/signup/validate-password")
    public ResponseEntity<?> validatePassword(
            @RequestBody PasswordRequest request
    ) {
        var response = userService.validatePassword(request.getPassword());
        return ResponseDto.ok(response);
    }
```